xml/templates/message_templates/participant_cancelled_html.tpl

```diff
@@ -34,16 +34,16 @@
      </tr>
      <tr>
       <td colspan="2" {$valueStyle}>
-       {$event.event_title}<br />
-       {$event.event_start_date|crmDate}{if $event.event_end_date}-{if $event.event_end_date|date_format:"%Y%m%d" == $event.event_start_date|date_format:"%Y%m%d"}{$event.event_end_date|crmDate:0:1}{else}{$event.event_end_date|crmDate}{/if}{/if}
+       {event.title}<br />
+       {event.start_date}{if $event.event_end_date}-{if $event.event_end_date|crmDate:"shortdate" == $event.event_start_date|crmDate:"shortdate"}{$event.event_end_date|crmDate:"Time"}{else}{$event.event_end_date|crmDate}{/if}{/if}
```

Comment:
also I think you need to convert the end_date as well

Reply:
@seamuslee001 I pulled out the event changes - the event tokens aren't working right at the moment - I think it was our last rc token loading change but will worry about it after these changes are resolved